.github/workflows/release.yml

```diff
@@ -171,8 +171,8 @@ jobs:
         run: |
           REF=${RELEASE_TAG:-${GITHUB_REF}}
           TAG=${REF##*/}
-          echo ::set-output name=ref::${REF}
-          echo ::set-output name=tag::${TAG}
+          echo "ref=${REF}" >> $GITHUB_OUTPUT
+          echo "tag=${TAG}" >> $GITHUB_OUTPUT
```

Comment:
Perhaps you need to specify `shell: bash`.
See also : https://github.com/google/go-github/pull/2492#issuecomment-1276469770